modules/ROOT/pages/rtc-supported-functionality.adoc

```diff
@@ -18,9 +18,13 @@ The Real-Time Collaboration (RTC) plugin supports the latest desktop versions of
 
 Real-Time Collaboration is currently not supported on mobile devices.
 
+== Input support
+
+The Real-Time Collaboration (RTC) plugin currently doesn't support Input Method Editor (IME) as a input method.
+
 == TinyMCE features (plugins) support when using RTC
 
-The Real-Time Collaboration (RTC) plugin supports a subset of the features that {productname} provides.
+Real-Time Collaboration does not currently support using an Input Method Editor (IME).
```

Comment:
Should this have been changed? This section doesn't appear to be about IME and we have it above already now. I think this would have been better left how it originally was.

Reply:
Seems like a mistake, perhaps Spocke copied my suggestion into the wrong place

Reply:
Ahh yeah, looks like 0b5e7ae339d41ca20d610811dc7ec1cb7ffe1c9f updated the wrong paragraph.

- [ ] Revert the change to this line and fix the correct paragraph.

Reply:
Yes, I did thanks for spotting this. Had to run of to a private meeting so was a bit stressed when I fixed that change.